README.md

```diff
@@ -44,6 +44,45 @@ Alternatively using `cargo install`:
     (while sleep 5; do echo; done) | target/release/metamath-knife --timing --jobs 4 --split --repeat --trace-recalc --verify set.mm/set.mm
     # then make small changes to the beginning, end, or middle of the DB and observe how behavior changes
 
+Here is the command line help, which gives an idea of the options available:
+```
+A Metamath database verifier and processing tool
+
+USAGE:
+    metamath-knife [FLAGS] [OPTIONS] <DATABASE>
+
+FLAGS:
+        --debug                Activates debug logs, including for the grammar building and statement parsing
+        --free                 Explicitly deallocates working memory before exit
+    -g, --grammar              Checks grammar
+    -h, --help                 Prints help information
+    -O, --outline              Shows database outline
+    -p, --parse-stmt           Parses all statements according to the database's grammar
+    -t, --parse-typesetting    Parses typesetting information
+    -F, --print-formula        Dumps the formulas of this database
+    -G, --print-grammar        Prints the database's grammar
+        --dump-typesetting     Shows typesetting information
+        --repeat               Demonstrates incremental verifier
+        --split                Processes files > 1 MiB in multiple segments
+        --timing               Prints milliseconds after each stage
```

Comment:
why not `--time` ?

(PS: I know all of these quibbles are not related to the current PR, but since I discovered metamath-knife only yesterday...)

Reply:
That was @sorear's original choice.  Both seem equally good to me.

Reply:
(I was also thinking of the command `time`, see `$ man time`)

Reply:
While `--timing` is plausible, `--time` seems preferable since it's similar to the `time` command.

Reply:
Ok, I changed the option to `--time`.